### PR TITLE
added docs for a live cluster

### PR DESCRIPTION
**Pull request by gtfisher**
Signed-off-by: gtfisher <grant@chaosiq.io>

### CODE REVIEW

File: docs/guides/live-cluster/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## Discover Reliability concerns from live Kubernetes Cluster
+
+To use Reliably to discover reliability concerns from your live cluster, you can
+run:
+
+```
+$ reliably discover --live
+```
+
+The command will generate output that will offer advice about Reliability concerns from your live cluster. An example for the generated output could be:
```
Comment (mperrien) [inline docs/guides/live-cluster/index.md:28]: What about
```suggestion
The command will generate an output that offers advice about reliability concerns from your live cluster. An example for the generated output could be:
```

File: docs/guides/live-cluster/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+The `reliably --live` command will connect and talk to your cluster using a
+ [kubeconfig][kubeconfig] file. This file typically lives at
+ `$HOME/.kube/config`. By default, `reliably --live` will use the kubeconfig
+ from the home location. You can override the path for the kubeconfig by adding
+ the `--kubeconfig` or `-k` flag:
+
+```
+$ reliably --live --kubeconfig=/reliably/kube-config
+```
+You can also use an Environemt Variable to locate your kubeconfig file by
```
Comment (mperrien) [inline docs/guides/live-cluster/index.md:49]:
```suggestion
You can also use an Environment Variable to locate your kubeconfig file by
```

File: docs/guides/live-cluster/index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+setting the `KUBECONFIG` Environemt Variable:
```
Comment (mperrien) [inline docs/guides/live-cluster/index.md:50]:
```suggestion
setting the `KUBECONFIG` Environment Variable:
```

File: docs/guides/live-cluster/index.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+title: Run with a live kubernetes cluster
+excerpt: Lorem Ipsum is simply dummy text.
+categories: ["guides"]
+status: published
+type: overview
+---
+# Run Reliably with a live Kubernetes Cluster
```
Comment (mperrien) [inline docs/guides/live-cluster/index.md:8]: Should it be "within"?

File: src/data/doc-guides-links.yaml [hunk 5]
```diff
@@ -1,5 +1,7 @@
 - title: Run as a kubectl Plugin
   link: /guides/kubectl-plugin/
+- title: Run with a live Kubernetes Cluster
```
Comment (mperrien) [inline src/data/doc-guides-links.yaml:3]: Same question as before, should it be "within", or "in"?

File: src/data/doc-root-links.yaml [hunk 6]
```diff
@@ -23,6 +23,9 @@
     - title: Run as a kubectl Plugin
       slug: kubectl-plugin
       link: /guides/kubectl-plugin/
+    - title: Run with a live Cluster
```
Comment (mperrien) [inline src/data/doc-root-links.yaml:26]: Same question, should it be "within", or "in"?

**Review by mperrien — COMMENTED** [review]
I made a few suggestions. I'll let you sort them and merge, @gtfisher

File: docs/guides/live-cluster/index.md (hunk 4 above)
Comment (gtfisher) [reply docs/guides/live-cluster/index.md:8]: @mperrien - I don't think so in this case, we are not running within the cluster in the current config

**Pull request merged**